sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -251,15 +270,46 @@ def _predict(self, fh, X=None):
         self.model.eval()
         from torch import from_numpy
 
-        hist = self._y.values.reshape((1, -1))
+        if isinstance(self._y.index, pd.MultiIndex):
+            lens = self._y.groupby(
+                level=list(range(len(self._y.index.levels) - 1))
+            ).apply(lambda x: len(x))
+            assert (lens == lens.iloc[0]).all(), "All series must has the same length"
```

Comment:
Why is this required?

Reply:
Not sure what you mean? No need to have all series the same length or simpler way to get the length? As we are going to convert it to numpy array, it's required to have all time series the same length right?

Reply:
From discussion: Check if all indexes are the same, since it probably would be very complicated to handle different cutoffs for the different time series.

Reply:
Done